hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -167,6 +169,11 @@
       .noDefaultValue()
       .withDocumentation("Key Generator class property for the hoodie table");
 
+  public static final ConfigProperty<HoodieTimelineZone> TIMELINE_TIMEZONE = ConfigProperty
+      .key("hoodie.table.timeline.zone")
+      .defaultValue(HoodieTimelineZone.LOCAL)
+      .withDocumentation("Set timeline timezone type");
```

Comment:
please describe the timezone in more detail, such as the usage.

Reply:
> please describe the timezone in more detail, such as the usage.

now change it to "User can set hoodie commit timeline timezone, such as utc, local. local is default"

Reply:
> please describe the timezone in more detail, such as the usage.

now change it to "User can set hoodie commit timeline timezone, such as utc, local. local is default"